anta/cli/get/utils.py

```diff
@@ -204,3 +211,49 @@ def create_inventory_from_ansible(inventory: Path, output: Path, ansible_group:
         raise ValueError(msg)
     ansible_hosts = deep_yaml_parsing(ansible_inventory)
     write_inventory_to_file(ansible_hosts, output)
+
+
+def explore_package(module_name: str, level: int = 0, test_name: str | None = None, *, short: bool = False) -> None:
+    """Parse submodules recursively and print AntaTest example."""
+    if (module_spec := importlib.util.find_spec(module_name)) is None or module_spec.origin is None:
+        return
+
+    if module_spec.submodule_search_locations:
+        for _, sub_module_name, ispkg in pkgutil.walk_packages(module_spec.submodule_search_locations):
+            qname = f"{module_name}.{sub_module_name}"
+            if ispkg:
+                explore_package(qname, level=level + 1, test_name=test_name)
+                continue
+            print_tests_examples(qname, level, test_name, short=short)
+
+    else:
+        print_tests_examples(module_spec.name, level, test_name, short=short)
+
+
+def print_tests_examples(qname: str, level: int, test_name: str | None, *, short: bool = False) -> None:
+    """Print tests in qname if matching test_name (or all if test_name is None.
+
+    TODO: Allow to give a package argument to import_module.
+    """
+    qname_module = importlib.import_module(qname)
+    module_printed = False
+    for _name, obj in inspect.getmembers(qname_module):
+        if not inspect.isclass(obj) or not issubclass(obj, AntaTest) or obj == AntaTest:
+            continue
+        if test_name and not obj.name.startswith(test_name):
```

Comment:
`startswith` could cause issue here:
```bash
(.dev) ~/git_projects/anta (anta-get-tests ✗) anta get tests --test VerifyNTP  
anta.tests.system:
  - VerifyNTP:
      # Verifies if NTP is synchronised.
  - VerifyNTPAssociations:
      # Verifies the Network Time Protocol (NTP) associations.
      ntp_servers:
        - server_address: 1.1.1.1
          preferred: True
          stratum: 1
        - server_address: 2.2.2.2
          stratum: 2
        - server_address: 3.3.3.3
          stratum: 2
```
We need exact match.

Reply:
yeah thats the only issue - we should rename this test ;)

Reply:
it is more useful to me to find all the tests with Tacacs in it for instance than to have to type them one by one